AbletonMCP_Remote_Script/AbletonMCP_Remote_Script/handlers/midi.py

```
@@ -0,0 +1,438 @@
+                    if hasattr(note, 'release_velocity'):
+                        note_dict["release_velocity"] = note.release_velocity
+                    notes.append(note_dict)
+                return {
+                    "clip_name": clip.name,
+                    "clip_length": clip.length,
+                    "note_count": len(notes),
+                    "extended": True,
+                    "notes": notes,
+                }
+            except Exception:
+                pass  # Fall through to legacy
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)